examples/tutorials/advanced/Federated Dataset.ipynb

```diff
@@ -0,0 +1,460 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h1>Federated Datasets</h1>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example demonstrates how you can develop your custom datasets that work with federated data loaders. The regular torch datasets do not always work directly with federated data loaders.The tutorial aims to highlight the differences and shows you how you could develop a federated dataset with your data that works with the federated data loader. Further you could also use Syft's Base dataset feature which simplify creating federated datasets.To demonstrate these we will show you how to load the SVHN (Street View House Numbers) dataset and convert it to a federated dataset.\n",
```

Comment:
There is "federated data loaders" many times in different sentences maybe...

Instead of:
> This example demonstrates how you can develop your custom datasets that work with federated data loaders. The regular torch datasets do not always work directly with federated data loaders.The tutorial aims to highlight the differences and shows you how you could develop a federated dataset with your data that works with the federated data loader. Further you could also use Syft's Base dataset feature which simplify creating federated datasets.

I would write something like:
*Unfortunately the regular torch datasets do not always work directly with federated data loaders. This example demonstrates how to develop a custom dataset that works with federated data loaders highlighting the differences. Further, you could also use Syft's Base dataset feature which simplifies creating federated datasets.*

Also, can you add a link to the official SVHN dataset?

Reply:
Thanks that simplified it :)